pkgs/development/libraries/protobuf/3.6.nix

```diff
@@ -0,0 +1,6 @@
+{ callPackage, lib, ... }:
+
+lib.overrideDerivation (callPackage ./generic-v3.nix {
```

Comment:
you should use overrideAttrs instead according to https://nixos.org/nixpkgs/manual/#sec-pkg-overrideDerivation